SmartEVSE-3/src/evse.cpp

```diff
@@ -2914,10 +2920,15 @@ void StartwebServer(void) {
         
         doc["home_battery"]["current"] = homeBatteryCurrent;
         doc["home_battery"]["last_update"] = homeBatteryLastUpdate;
-        
-        doc["ev_meter"]["EVMeter_import_active_energy"] = round(EnergyEV / 100)/10; //in kWh, precision 1 decimal
-        doc["mains_meter"]["MainsMeter_import_active_energy"] = round(Mains_import_active_energy / 100)/10; //in kWh, precision 1 decimal
-        doc["mains_meter"]["MainsMeter_export_active_energy"] = round(Mains_export_active_energy / 100)/10; //in kWh, precision 1 decimal
+
+        doc["ev_meter"]["description"] = EMConfig[EVMeter].Desc;
+        doc["ev_meter"]["address"] = EVMeterAddress;
+        doc["ev_meter"]["power"] = round(PowerMeasured / 100)/10; //in kWh, precision 1 decimal
+        doc["ev_meter"]["total_kwh"] = round(EnergyEV / 100)/10; //in kWh, precision 1 decimal
+        doc["ev_meter"]["charged_kwh"] = round(EnergyCharged / 100)/10; //in kWh, precision 1 decimal
+
+        doc["mains_meter"]["import_active_energy"] = round(Mains_import_active_energy / 100)/10; //in kWh, precision 1 decimal
```

Comment:
Great improvement to the names I chose, hadn't noticed the double "mains meter" reference, thanks!